### PR TITLE
Add University of Wisconsin/EHT/EHT:EHT-EP (SOFTWARE-5283)

### PR DESCRIPTION
**Pull request by edquist**
please scrutinize!

### CODE REVIEW

File: topology/University of Wisconsin/EHT/EHT.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          Name: Carl Edquist
+      Security Contact:
+        Primary:
+          ID: OSG1000003
+          Name: Brian Lin
+        Secondary:
+          ID: OSG1000002
+          Name: Matyas Selmeci
+    Description: OSPool A100 for EHT
+    FQDN: tiger2002.chtc.wisc.edu
```
Comment (brianhlin) [inline topology/University of Wisconsin/EHT/EHT.yaml:27]: Let's give it a nicer fake FQDN (maybe this ends up being a different node in the future): eht-ep.chtc.wisc.edu maybe

File: topology/University of Wisconsin/EHT/EHT.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+GroupDescription: EHT OSPool Resources
+GroupID: 1303
+Production: true
+SupportCenter: Self Supported
+Resources:
+  EHT-EP:
```
Comment (brianhlin) [inline topology/University of Wisconsin/EHT/EHT.yaml:6]:
```suggestion
  CHTC-EHT-EP:
```

**Review by brianhlin — APPROVED** [review]
Throw this into `UW/CHTC/CHTC_OSPOOL.yaml` instead. The CHTC is still contributing these hours through the OSPool, just specifically for the EHT project. Pre-approving since I'll be OOO starting tomorrow

File: topology/University of Wisconsin/EHT/EHT.yaml (hunk 1 above)
Comment (edquist) [reply topology/University of Wisconsin/EHT/EHT.yaml:27]: :+1:

File: topology/University of Wisconsin/EHT/EHT.yaml (hunk 2 above)
Comment (edquist) [reply topology/University of Wisconsin/EHT/EHT.yaml:6]: :+1:

File: topology/University of Wisconsin/EHT/EHT.yaml (hunk 1 above)
Comment (edquist) [reply topology/University of Wisconsin/EHT/EHT.yaml]: updated

File: topology/University of Wisconsin/EHT/EHT.yaml (hunk 2 above)
Comment (edquist) [reply topology/University of Wisconsin/EHT/EHT.yaml]: updated

**Comment by edquist** [comment]
>Throw this into UW/CHTC/CHTC_OSPOOL.yaml instead

done

**Pull request merged**